.github/workflows/build_bitacross.yml

```diff
@@ -0,0 +1,52 @@
+name: Build bitacross-worker
+
+on:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  build-worker:
+    runs-on: tee-prod-builder
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Build local builder
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: bitacross-worker/build.Dockerfile
+          tags: local-builder:latest
+          target: builder
+          build-args: |
+            WORKER_MODE_ARG=offchain-worker
+            SGX_MODE=HW
+            ADDITIONAL_FEATURES_ARG=      
+            IMAGE_FOR_RELEASE=true
+
+      - name: Build worker
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: bitacross-worker/build.Dockerfile
+          tags: litentry/bitacross-worker:latest
```

Comment:
I suggest that the docker tag is parameterized too

Reply:
It'll use `input.ref` for both checking out the code and tagging image

Reply:
tags should be `${{ github.event.inputs.ref || github.ref }}` instead of latest?

Reply:
I would like to test this workflow now but seems I cannot start it from GitHub.